code/modules/xenomorph/xeno_structures.dm

```diff
@@ -1235,7 +1238,7 @@ TUNNEL
 		return FALSE
 
 	if(X.a_intent == INTENT_HARM && isxenodrone(X))
-		to_chat(X, span_xenowarning("We uproot [src]!"))
+		balloon_alert(X, "Uprooted [src]")
```

Comment:
A lot of the balloon alerts for the fruits below this point use name inserts ([src]) reword those pls